airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/demo/file_based/platform/CsvV2RecordWriter.java

```diff
@@ -0,0 +1,31 @@
+package io.airbyte.integrations.destination.snowflake.demo.file_based.platform;
+
+import io.airbyte.protocol.models.v0.AirbyteRecordMessage;
+import java.util.UUID;
+
+/**
+ * Other implementations exist, e.g. avro, parquet, jsonl. Platform would provide generic implementations,
+ * though destinations could provide their own implementations if they want to do something fancy.
+ * <p>
+ * This implementation would write the DV2 CSV format (raw_id, extracted_at, data, loaded_at; loaded_at is always null).
+ */
+public class CsvV2RecordWriter implements RecordWriter {
```

Comment:
These maybe belong in destinations-land. `CsvV2` doesn't feel like a platform concern.

Reply:
I think they do! We want fast and reusable file-writers, no?  They will always be the same I think.